server/routes/validators/reviewPlan/reviewNoteValidator.ts

```diff
@@ -0,0 +1,12 @@
+import type { ReviewNoteForm } from 'reviewPlanForms'
+import formatErrors from '../../errorFormatter'
+
+export default function validateReviewNote(reviewNoteForm: ReviewNoteForm): Array<Record<string, string>> {
+  const errors: Array<Record<string, string>> = []
+
+  if (!reviewNoteForm.notes) {
```

Comment:
Whilst the ticket doesn't explicitly mention it, the parent epic links to the [error content](https://docs.google.com/spreadsheets/d/1VPYH0r2Yo4WeYACwVeiTkvZFYXsNfJhGeOwV-mmSCso/edit?usp=sharing) requirements, and within there [I think it specifies that the max length for this field is 512 characters](https://docs.google.com/spreadsheets/d/1VPYH0r2Yo4WeYACwVeiTkvZFYXsNfJhGeOwV-mmSCso/edit?gid=851177864#gid=851177864&range=D75:F75)

Could you implement that in this validator, and also a unit test for the validator class? (have a look at the tests for the other validator classes and follow suit 😁 👍 )